src/main/java/gtPlusPlus/core/item/bauble/FireProtectionBauble.java

```diff
@@ -0,0 +1,137 @@
+package gtPlusPlus.core.item.bauble;
+
+import static gregtech.api.enums.Mods.GTPlusPlus;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.UUID;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.potion.Potion;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraft.world.World;
+
+import baubles.api.BaubleType;
+import cpw.mods.fml.common.registry.GameRegistry;
+import gtPlusPlus.core.creative.AddToCreativeTab;
+import gtPlusPlus.core.util.reflect.ReflectionUtils;
+import gtPlusPlus.preloader.DevHelper;
+
+public class FireProtectionBauble extends BaseBauble {
+
+    public static HashMap<UUID, Boolean> mDataMap = new HashMap<UUID, Boolean>();
+    public static HashSet<UUID> mPlayerMap = new HashSet<UUID>();
```

Comment:
Is this set used anywhere?

Reply:
I have no idea, probably not, but this is purely an undo of Alks original code.